card.py

```diff
@@ -2,7 +2,28 @@ class Card:
     def __init__(self, v, s):
         self.value = v
         self.suit = s
+        self.score = self.get_score()
 
     def __repr__(self):
-        return f'{self.value} of {self.suit}'
+        return f'{self.get_value_name()}{self.suit}'
 
+    def get_score(self):
+        if self.value > 10:
+            return 10
+        elif self.value == 1:
+            return 11
+        else:
+            return self.value
+
+    def get_value_name(self):
```

Comment:
It might be worth making this a property. 

```python
@property
def value_name(self):
    ...
```

Then, you can call `self.value_name` to get this. No need for namespacing something with `get_` if it doesn't take any arguments other than `self`. That's what properties are around to make easier.